COMETwebapp/Components/Canvas/ActualFiniteStateSelectorComponent.razor.cs

```diff
@@ -0,0 +1,122 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="StateSelector.razor.cs" company="RHEA System S.A.">

```

Comment:
The copyright filename is not aligned with the current filename. 
General remarks, the copyright year can move to 2023 no?